### ISSUE TITLE
Not an issue. but enhancement maybe

### ISSUE THREAD

**Issue by Norwayman** [septor/dateplus#3]
Could this be integrated with Google Calendar so that users could subscribe to the calendar and it would appear in their Google Calendar?  Just an idea :)

**Comment by septor** [comment]
Hmm, this is kind of interesting.

Since Google Calendar already has holidays, I suspect you are referring to Userdays?

I like this idea. I'll mull this over a bit and do some research.

**Comment by Norwayman** [comment]
Yes, I suppose it has to be Userdays.
If not the holidays would appear twice in Google Calendar, no?

**Comment by septor** [comment]
Yeah, they would. I might have time to do some research on this today.

**Comment by septor** [comment]
I made a post on Google+ to hopefully get some feedback on this.

I did some looking yesterday and I still have some kinks to work out before I can get some serious testing done and before I can start implementing this into Date+, but I'm looking forward to getting this added in.

**Comment by septor** [comment]
Just an update (if you're tracking this):

I've managed to successfully pull calendar data (it, so far as I can tell, pulls all the calendar data from the user's account, so that's pretty great) and display it (not in e107, but it's easy enough that it won't be too difficult to port over).

The problem I'm having right now (is getting enough time to sit down and work on this for a solid stretch of time!!!!) is inserting a new event into a calendar. I'm going to scrap most of my code and start fresh, since I think I'm mucking something up with so much going on!

So basically, I'm in a much better place with this than the previous post!

**Comment by Norwayman** [comment]
That is great news, man! :)
Do the stuff when you have time. Remeber, this is just a hobby! :)
Looking forward to the progress.

**Comment by septor** [comment]
Good news, I have successfully created a new event with the code!

Now I just need to do some further testing and then I'll start pushing the code to the plugin.

**Comment by septor** [comment]
Alright! I'm ready to start merging the code into the plugin.

Should this replace the current Userday system or be an additional thing?

**Comment by Norwayman** [comment]
I think maybe it should be optional. Never good to force anyone. What do
you think?
7. jan. 2016 15:04 skrev "Patrick Weaver" notifications@github.com:

**Comment by septor** [comment]
I think it should be an additional, optional, addition.

I'm really considering just making a plugin that utilizes Google Calendar instead of adding the stuff into this plugin since you're going to either use the features of Date+ or you're going to use the Google Calendar stuff (which is far and away a better option, imo).

No point in having the overhead of the other features if you'll never use them, you know?

Thoughts?

**Comment by Norwayman** [comment]
Yes, I definately think I agree with you :)
